hashes/src/sha256t.rs

```diff
@@ -31,16 +31,19 @@ pub trait Tag {
 }
 
 /// Output of the SHA256t hash function.
-#[cfg_attr(feature = "schemars", derive(schemars::JsonSchema))]
 #[repr(transparent)]
-pub struct Hash<T: Tag>(
-    #[cfg_attr(
-        feature = "schemars",
-        schemars(schema_with = "crate::util::json_hex_string::len_32")
-    )]
-    [u8; 32],
-    #[cfg_attr(feature = "schemars", schemars(skip))] PhantomData<T>,
-);
+pub struct Hash<T: Tag>([u8; 32], PhantomData<T>);
+
+#[cfg(feature = "schemars")]
```

Comment:
docsrs attribute would be nice

Reply:
Oh I understand now, the second comment negates the first. Damn, I'm slow sometimes. So no action required here. I thought these comments were the source of your missing ACK.

Reply:
Unless I was hallucinating the docs attribute was there at the time I wrote the second comment.